driver/src/test/java/org/neo4j/driver/util/Neo4jRunner.java

```diff
@@ -83,9 +83,10 @@ public Neo4jRunner() throws IOException
         {
             if ( neo4jHome.exists() )
             {
-                System.out.println( "Found an old Neo4j server in: " + neo4jHome.getAbsolutePath() + ". Deleting to " +
-                                    "use a new one." );
-                FileTools.deleteRecursively( neo4jHome );
+                return;
+//                System.out.println( "Found an old Neo4j server in: " + neo4jHome.getAbsolutePath() + ". Deleting to " +
+//                                    "use a new one." );
+//                FileTools.deleteRecursively( neo4jHome );
```

Comment:
Oh, this was actually by accident - I just added it temporarily to speed up the test. It makes sense to have, I think, but it'd be nice to be able to turn it off as well, or to have it check if the downloaded file it has is the same as the file thats up remotely. Wonder if we can get S3 to give us 304-not-modified responses..

In any case, it's a separate thing, I'll undo this.